package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "spruce",
-  "version": "0.7.1",
+  "version": "0.8.0",
```

Comment:
Didnt realize this didnt push up to upstream when you deployed.

Reply:
was confused by this

Reply:
When we issue a new release this version number gets increased it coincides with the released versions. https://github.com/evergreen-ci/spruce/releases

Its used by npm to determine what the next version of our project should be for the next time we release.